token-swap/js/cli/token-swap-test.js

```diff
@@ -37,14 +38,25 @@ let tokenAccountB: PublicKey;
 // curve type used to calculate swaps and deposits
 const CURVE_TYPE = CurveType.ConstantProduct;
 // Initial amount in each swap token
-const BASE_AMOUNT = 1000;
+let currentSwapTokenA = 1000;
```

Comment:
There's a lot of state sharing in these tests, which has gotten more confusing with fees